oat-sa/extension-tao-itemqti
# Feature/replicate item tree structure into media manager

This PR adds the possibility to specify several levels of tree structure to create in the media manager in one call.
The idea behind is to be able to reproduce the tree structure of the item bank in which a set of items with a shared stimulus is imported.
When we import a package with shared stimulus in this place:
![item-tree-structure](https://user-images.githubusercontent.com/47318116/78377734-4638db00-75d0-11ea-9941-cfd65321b808.png)
it creates the similar tree structure to store the stimulus in mediamanager:
![media-tree-structure](https://user-images.githubusercontent.com/47318116/78377753-4cc75280-75d0-11ea-81e8-9d310bcc4563.png)
And finally adds the label of the first item imported as a last tree level.

Requires: 
- https://github.com/oat-sa/generis/pull/763
- https://github.com/oat-sa/extension-tao-mediamanager/pull/151

To test the behaviour, you need:
- a Tao installation with MadiaManager extension
- this sample QTI package with shared stimulus:
[casus_1.zip](https://github.com/oat-sa/extension-tao-itemqti/files/4451617/casus_1.zip)
- go to item bank, create the class(es) that you 'd like to reproduce in MediaManager
- import the package.
It should create 4 items named `Conseiller 001 - Réaction 1` to `Conseiller 001 - Réaction 4`, and in the media manager, the corresponding classes and finally the last class `Conseiller 001 - Réaction 1` named after the label of the first imported item, and in this class, a `shared.xml` file.
